### DISCUSSION TITLE
关于 Stream Load 返回结果的疑问

### DISCUSSION THREAD

**Discussion by gilboom** [apache/doris#24991] [RFC]
请问一下，`LoadTimeMs`、`ReadDataTimeMs `、`WriteDataTimeMs` 这几个字段的关系是？

我在使用 `org.apache.doris.flink.tools.cdc.CdcTools` 这个工具去使用 Flink CDC 同步 MySQL。

我的理解是，`LoadTimeMs` 从 Flink TaskManager 写入一批数据到 Doris BE 的时间，`ReadDataTimeMs` 是数据从 Flink TaskManager 传输到 Doris BE 的时间，`WriteDataTimeMs` 是数据到达 Doris BE 之后，写入到硬盘的时间，不知道理解的是否正确？
 
![image](https://github.com/apache/doris/assets/23697209/a00c6f6d-8b09-4a3a-bf77-b4af4d0b48cb)

**Comment by yagagagaga** [comment]
You can think so.

**Reply by gilboom** [reply]
谢谢，但是我通过观察日志的时间，看起来并不需要 9142ms 的时间去完成数据从 Flink 加载到 Doris，是我理解的有误吗？

**Reply by liujunfei2230** [reply]
请问下那这里的数据可见，是在readdata后，还是writedata后？

**Reply by yagagagaga** [reply]
stream load 的 respond 返回后，数据立马可见
